### ISSUE TITLE
A11y Fail: Alert - Add aria-label for icon.

### ISSUE THREAD

**Issue by olehnoskov-cengage** [cengage/react-magma#1766]
**Describe the bug №1**
There is an icon rendered as an <svg> that is announced by screen readers as an unlabelled image. 
This may create uncertainty for screen reader users who may wonder what the purpose of the image is.

**Expected behavior**
Please add role="img" and an aria-label to the <svg> so that it is described by screen readers:
 - as a "info icon" -> for Info,
 -  as a "success icon" -> for Success,
 -  as a "warning icon" -> for Warning,
 - as a "danger icon" -> for Danger.

**Screenshots**

![Image](https://github.com/user-attachments/assets/09e02a36-e11d-4520-a329-8b0e605e251e)

**Additional context**
[Linked spike](https://github.com/cengage/react-magma/issues/1748)

[Original bug](https://jira.cengage.com/browse/NG-96812)

[Providing alternative text for informative images](https://perkins.caspio.com/dp/ca3640008726de167a974caaba70?Advice_ID=ADVICE_004)
